alembic/versions/2024_05_22_2011-2ea9f6f0b790_make_wcs_a_fileondiskmixin.py

```diff
@@ -0,0 +1,46 @@
+"""make WCS a FileOnDiskMixin
+
+Revision ID: 2ea9f6f0b790
+Revises: ec64a8fd8cf3
+Create Date: 2024-05-22 20:11:31.195961
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '2ea9f6f0b790'
+down_revision = '9a4097979249'
```

Comment:
Before you merge, make sure the last revision on "main" has a hash that matches the down revision. If some one else has merged a migration you will need to update this.